File: Python/UberBlack.py
from car import Car


class UberBlack(Car):
    typerCarAccepter = []
    seatsMaterial    = []
    
    
    def __init__(self,license, driver, typerCarAccepter, seatsMaterial):
        super().__init__(license, driver)
        self.typerCarAccepter = typerCarAccepter
        self.seatsMaterial = seatsMaterial

File: java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack  extends Car{
    Map<String,  Map<String, Integer>> typerCarAccepter;
    ArrayList<String> seatsMaterial;



    public UberBlack (String license, Account driver,Map<String,  Map<String, Integer>> typerCarAccepter,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typerCarAccepter = typerCarAccepter;
        this.seatsMaterial = seatsMaterial;
    }
    
}
